src/components/Messages/Message/index.js

```diff
@@ -141,5 +142,6 @@ export const Message = memo(({
 			})}
 		</MessageContent>
 		{!compact && !message.t && <MessageTime normal={!me} inverse={me} ts={ts} />}
+		{message.t === 'webRTC_call_started' && message.callStatus === 'accept' ? <ShowCallTime stime={ts} /> : null}
```

Comment:
This should be changed to system messages defined [here](https://github.com/RocketChat/Rocket.Chat.Livechat/pull/617/files#diff-bb3347eeb1e2807319f728d4892a5edba39d39eab105e29fd6c120d2820023e1R84). Also it seems here we're not handling the `jitsi_call_started` event, so we'll need to handle this as well here